### DISCUSSION TITLE
如何更换主题中使用.jsdelivr的资源？

### DISCUSSION THREAD

**Discussion by luluxiaoyu** [vuepress-theme-hope/vuepress-theme-hope#4519] [Q&A] [answered]
如何更换主题中使用.jsdelivr的资源？
这个有时候会被屏蔽 想换其他的cdn

**Comment by Mister-Hope** [comment, marked as answer]
所有的链接都不是写死的。自己检查文档，如果遇到问题，带着你不会改的部分的完整详情留言。

**Reply by luluxiaoyu** [reply]
> 所有的链接都不是写死的。自己检查文档，如果遇到问题，带着你不会改的部分的完整详情留言。

https://github.com/orgs/vuepress-theme-hope/discussions/4519  https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6/js/solid.min.js  https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6/js/fontawesome.min.js  我想换成其他的cdn

**Reply by Mister-Hope** [reply]
这个你不要用关键词自己换一个cdn的就行

**Reply by luluxiaoyu** [reply]
> 这个你不要用关键词自己换一个cdn的就行

  iconAssets: ["https://cdn.bootcdn.net/ajax/libs/font-awesome/6.6.0/js/fontawesome.min.js",
"https://cdn.bootcdn.net/ajax/libs/font-awesome/6.6.0/js/brands.min.js",
"https://cdn.bootcdn.net/ajax/libs/font-awesome/6.6.0/js/solid.min.js"], 我这样写的 但是图标似乎不显示了

**Reply by Mister-Hope** [reply]
那是因为你这个不是标准的CDN链接

你自己手动设一下iconPrefix

**Reply by luluxiaoyu** [reply]
> 那是因为你这个不是标准的CDN链接
> 
> 你自己手动设一下iconPrefix

可以了 谢谢！
